test/FirstDerivative/ClassMethodCall.C

```diff
@@ -102,5 +102,11 @@ int main () {
   printf("Result is = %f\n", vm_darg0_A.execute(a, 2, 3)); // CHECK-EXEC: Result is = 1.0000
   auto vm_darg0_B = clad::differentiate(&B::vm, 0);
   printf("Result is = %f\n", vm_darg0_B.execute(b, 2, 3)); // CHECK-EXEC: Result is = 4.0000
+  printf("%s", vm_darg0_B.getCode()); //CHECK-EXEC:   float vm_darg0(float x, float y) { 
+                                      //CHECK-EXEC-NEXT:       float _d_x = 1;
+                                      //CHECK-EXEC-NEXT:       float _d_y = 0;
+                                      //CHECK-EXEC-NEXT:       return _d_x * x + x * _d_x + _d_y * y + y * _d_y;
+                                      //CHECK-EXEC-NEXT:   }
+
```

Comment:
```suggestion
  printf("%s\n", vm_darg0_B.getCode());
  //CHECK-EXEC:   float vm_darg0(float x, float y) { 
  //CHECK-EXEC-NEXT:       float _d_x = 1;
  //CHECK-EXEC-NEXT:       float _d_y = 0;
  //CHECK-EXEC-NEXT:       return _d_x * x + x * _d_x + _d_y * y + y * _d_y;
  //CHECK-EXEC-NEXT:   }
```

We should add a test for the `<invalid>` case. It should be something like:

```cpp
clad::CladFunction invalid_func;
printf("%s\n", invalid_func.getCode());

CHECK-EXEC-NEXT: <invalid>
```